src/viresh/assignment1/HtmlFormOperations.java

```diff
@@ -0,0 +1,72 @@
+package viresh.assignment1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class HtmlFormOperations {
+
+	public static void main(String[] args) throws InterruptedException {
+		System.setProperty("webdriver.chrome.driver", ".//resources/windows/chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
+		driver.get(
+				"file:///C:/Users/Viresh.jangam/OneDrive%20-%20Wolters%20Kluwer/Desktop/Viresh/TechnoCredits/Assignments/Selenium/Assignment1/SeleniumAssignment_1.html");
+
+		// adding a value into textbox
+		System.out.println("The browser name is: " + driver.getTitle());
+		driver.findElement(By.id("first name")).sendKeys("Viresh");
+		System.out.println("first name is: " + driver.findElement(By.id("first name")).getAttribute("value"));
```

Comment:
If you want to perform multiple action on webElement then store in any variable. Do not fine one element multiple time.

Reply:
Okay Krishna, Noted. Thanks.